week 5/README.md

```diff
@@ -190,3 +190,25 @@ Output
 1 2 3
 ```
 
+
+### Task 3 - WTF Cipher
```

Comment:
1. What you are describing is known as [Vigenere's cipher](https://en.wikipedia.org/wiki/Vigen%C3%A8re_cipher)
2. Let's also add a decoding part; it's not very interesting otherwise.